features/Context/FieldTypeFormContext.php

```diff
@@ -69,13 +69,13 @@ public function iEditOrCreateContentOfThisType()
     }
 
     /**
-     * @Then /^the edit form should contain a fieldset named after the field definition$/
+     * @Then /^the edit form should contain a container div labelled after the field definition$/
```

Comment:
If we wanted to be a bit more future proof, we could find a phrasing that doesn't refer to the technical elements. 
What about `Then the edit form should contain an identifiable widget for that field definition`.

The idea is to say that it should show up. How we check it is up to us. The "identifiable" part is to make sure that we can identify it from a frontend api perspective.

What do you think ?